docs/concepts/Dialog-Command.md

```diff
@@ -0,0 +1,78 @@
+# Dialog Command
+
+`askx dialog` opens a interactive mode, similar to Node's REPL, in which developers can simulate a multi-turn conversation with  Alexa. For each utterance, Dialog command leverages [simulate-skill](https://developer.amazon.com/en-US/docs/alexa/smapi/ask-cli-command-reference.html#simulate-command) to asynchronously poll simulation result until it's available and shows Alexa text responses. **Note:** Skill must have been enabled for simulation before running the dialog command.
+
+Structure of the dialog command is:
+
+`askx dialog [-s|--skill-id <skill-id>] [-l|--locale <locale>] [-g|--stage <stage>] [-r|--replay <file-path>] [-p|--profile <profile>] [--debug]`
+
+Default value for `--profile`is `default`and `--stage` is `development`. Specifying `--debug` appends a debug message to the standard error.
```

Comment:
We need to show the option description using a list. In the list you can describe the default value, or enum values, or environmental variable usage (ASK_DEFAULT_DEVICE_LOCALE for locale and ASK_DEFAULT_PROFILE for profile) or brief description.